docs/skopeo-copy.1.md

```diff
@@ -80,6 +84,25 @@ To copy and sign an image:
 $ skopeo copy --sign-by dev@example.com atomic:example/busybox:streaming atomic:example/busybox:gold
 ```
 
+To encrypt an image:
+```sh
+./skopeo copy docker://docker.io/library/nginx:latest oci:local_nginx:latest
+
+openssl genrsa -out private.key 1024
+openssl rsa -in private.key -pubout > public.key
+
+./skopeo  copy --encryption-key jwe:./public.key oci:local_nginx:latest oci:try-encrypt:enc
```

Comment:
Can we name the destination `oci:local_nginx:encrypted`? I think this will highlight the relationship with the source a bit more. Same for the commands below.